src/taipy/core/_entity/_labeled.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2023 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+from typing import Optional
+
+
+class _Labeled:
+    __LABEL_SEPARATOR = ">"
+
+    def get_label(self) -> str:
+        return self._get_explicit_label() or self._generate_label()
+
+    def get_simple_label(self) -> str:
+        if explicit_label := self._get_explicit_label():
+            return explicit_label
+        return self._generate_label(True)
```

Comment:
```suggestion
        return self._get_explicit_label() or self._generate_label(True)
```
This section should be similar to the `get_label()` right?